### PR TITLE
Pluralize Controller Names

### PR DESCRIPTION
**Pull request by apanzerj**
### PR checklist
- [x] Read the [contribution guildelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [x] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.
### Description of the PR

So I missed one more spot where we needed to pluralize names. In addition this fixes a glitch on the readme.

### CODE REVIEW

**Comment by apanzerj** [comment]
@wing328 can you take a look?

**Comment by apanzerj** [comment]
* added: fix the bang to hash bang in ./bin/rails5-petstore-server.sh

**Comment by apanzerj** [comment]
arg. got some garbage in there

**Comment by wing328** [comment]
cc @zlx

File: modules/swagger-codegen/src/main/resources/rails5/routes.mustache [hunk 1]
```diff
@@ -12,7 +12,7 @@ Rails.application.routes.draw do
 {{#apis}}
 {{#operations}}
 {{#operation}}
-  add_swagger_route '{{httpMethod}}', '{{basePathWithoutHost}}{{path}}', controller_name: '{{classVarName}}', action_name: {{#isRestfulIndex}}'index'{{/isRestfulIndex}}{{#isRestfulCreate}}'create'{{/isRestfulCreate}}{{#isRestfulUpdate}}'update'{{/isRestfulUpdate}}{{#isRestfulShow}}'show'{{/isRestfulShow}}{{#isRestfulDestroy}}'destroy'{{/isRestfulDestroy}}{{^isRestful}}'{{nickname}}'{{/isRestful}}
+  add_swagger_route '{{httpMethod}}', '{{basePathWithoutHost}}{{path}}', action_name: {{#isRestfulIndex}}'index'{{/isRestfulIndex}}{{#isRestfulCreate}}'create'{{/isRestfulCreate}}{{#isRestfulUpdate}}'update'{{/isRestfulUpdate}}{{#isRestfulShow}}'show'{{/isRestfulShow}}{{#isRestfulDestroy}}'destroy'{{/isRestfulDestroy}}{{^isRestful}}'{{nickname}}'{{/isRestful}}
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/rails5/routes.mustache]: @apanzerj did you have a chance to review the feedback by @zlx?

**Comment by apanzerj** [comment]
Yeah, nevermind. I made some mistakes of my own :-O

**Pull request closed without merging**